src/Label2.tsx

```diff
@@ -0,0 +1,105 @@
+import React from 'react'
+import styled from 'styled-components'
+import {variant} from 'styled-system'
+import sx, {SxProp, BetterSystemStyleObject} from './sx'
+import {get} from './constants'
+
+export type LabelProps = {
+  /** The color of the label */
+  variant?: LabelColorOptions
+  /** How large the label is rendered */
+  size?: LabelSizeKeys
+} & SxProp
+
+export type LabelColorOptions =
+  | 'default'
+  | 'primary'
+  | 'secondary'
+  | 'accent'
+  | 'success'
+  | 'attention'
+  | 'severe'
+  | 'danger'
+  | 'done'
+  | 'sponsors'
+
+type LabelSizeKeys = 'small' | 'large'
+
+export const variants: Record<
+  LabelColorOptions,
+  {
+    borderColor: React.CSSProperties['color']
+    color?: React.CSSProperties['color']
+  }
+> = {
+  default: {
+    borderColor: 'border.default'
+  },
+  primary: {
+    borderColor: 'fg.default'
+  },
+  secondary: {
+    borderColor: 'border.muted',
+    color: 'fg.muted'
+  },
+  accent: {
+    borderColor: 'accent.emphasis',
+    color: 'accent.fg'
+  },
+  success: {
+    borderColor: 'success.emphasis',
+    color: 'success.fg'
+  },
+  attention: {
+    borderColor: 'attention.emphasis',
+    color: 'attention.fg'
+  },
+  severe: {
+    borderColor: 'severe.emphasis',
+    color: 'severe.fg'
+  },
+  danger: {
+    borderColor: 'danger.emphasis',
+    color: 'danger.fg'
+  },
+  done: {
+    borderColor: 'done.fg',
+    color: 'done.emphasis'
+  },
+  sponsors: {
+    borderColor: 'sponsors.fg',
+    color: 'sponsors.emphasis'
+  }
+}
+
+const sizes: Record<LabelSizeKeys, BetterSystemStyleObject> = {
+  small: {
+    height: '20px',
+    padding: '0 7px' // hard-coded to align with Primer ViewCompnents and Primer CSS
+  },
+  large: {
+    height: '24px',
+    padding: '0 10px' // hard-coded to align with Primer ViewCompnents and Primer CSS
+  }
+}
+
+export const Label = styled.span<LabelProps>`
+  align-items: center;
+  background-color: transparent;
+  border-width: 1px;
+  border-radius: 999px;
```

Comment:
Should we use relative units like PVC too? `2em`

Reply:
The border-radius isn't relative to anything - it's just a really high number to make it perfectly round.

border-width is also not relative to anything, it's just 1px.